tests/compilation/test_predictor_rl.py

```diff
@@ -44,7 +44,9 @@ def test_qcompile_with_newly_trained_models() -> None:
     if not model_path.exists():
         with pytest.raises(
             FileNotFoundError,
-            match=re.escape("The RL model is not trained yet. Please train the model before using it."),
+            match=re.escape(
+                "The RL model model_expected_fidelity_ionq_harmony is not trained yet. Please train the model before using it."
```

Comment:
```suggestion
                "The RL model 'model_expected_fidelity_ionq_harmony' is not trained yet. Please train the model before using it."
```
Could use some quotes to be a little more readable.

Reply:
Agreed. I added it.